metrics/prometheus_machine.go

```diff
@@ -192,7 +193,29 @@ func NewPrometheusMachineCollector(i infoProvider, includedMetrics container.Met
 			},
 		}...)
 	}
+	if includedMetrics.Has(container.VmStatMetrics) {
+		c.machineMetrics = append(c.machineMetrics, []machineMetric{{
+			name:        "machine_vmstat",
+			help:		 "Various /proc/vmstat metrics",
```

Comment:
please correct help to more descriptive form, e.g. "Virtual memory statistics" - it comes form documentation https://man7.org/linux/man-pages/man5/proc.5.html